client/blocks/site/style.scss

```diff
@@ -89,7 +89,7 @@
 	overflow: hidden;
 	white-space: nowrap;
 	&::after {
-		@include long-content-fade();
+		@include long-content-fade($color: lighten( $sidebar-bg-color, 5% ));
```

Comment:
We need to do this from the sidebar to avoid messing other contexts where site is displayed: `.sidebar .site`.